scripts/modules/aws-mws-common/kms.tf

```diff
@@ -0,0 +1,26 @@
+variable "databricks_aws_account_id" {}
```

Comment:
can we add a default value for databricks aws account id?... it's granting databricks prod account to encrypt/decrypt with that key and there's only one account. no it doesn't make sense to have it always set for tests.

Reply:
resolved